Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -53,8 +53,11 @@ def check_xcode_up_to_date
         return unless MacOS::Xcode.installed?
         return unless MacOS::Xcode.outdated?
 
-        # CI images are going to end up outdated so don't complain.
-        return if ENV["TRAVIS"] || ENV["CIRCLECI"]
+        # Travis CI images are going to end up outdated so don't complain when
+        # `brew test-bot` runs `brew doctor` in the CI for the Homebrew/brew
+        # repository. This only needs to support whatever CI provider
+        # Homebrew/brew is currently using.
+        return if ENV["TRAVIS"]
```

Comment:
Note that Linuxbrew currently does import `extend/os/mac` because we still use some functions in there. That definitely needs to be fixed. Linuxbrew should not import  `extend/os/mac`.